diff -ur fontconfig-2.11.1.orig/fontconfig/fontconfig.h fontconfig-2.11.1/fontconfig/fontconfig.h
--- fontconfig-2.11.1.orig/fontconfig/fontconfig.h	2017-11-27 19:46:28.708675323 -0800
+++ fontconfig-2.11.1/fontconfig/fontconfig.h	2017-11-27 19:47:14.172539462 -0800
@@ -123,7 +123,8 @@
 #define FC_USER_CACHE_FILE	    ".fonts.cache-" FC_CACHE_VERSION
 
 /* Adjust outline rasterizer */
-#define FC_CHAR_WIDTH	    "charwidth"	/* Int */
+#define FC_CHARWIDTH	    "charwidth"	/* Int */
+#define FC_CHAR_WIDTH	    FC_CHARWIDTH
 #define FC_CHAR_HEIGHT	    "charheight"/* Int */
 #define FC_MATRIX	    "matrix"    /* FcMatrix */
 
diff -ur fontconfig-2.11.1.orig/src/fcobjs.h fontconfig-2.11.1/src/fcobjs.h
--- fontconfig-2.11.1.orig/src/fcobjs.h	2017-11-27 19:46:28.712675311 -0800
+++ fontconfig-2.11.1/src/fcobjs.h	2017-11-27 19:47:14.172539462 -0800
@@ -51,7 +51,7 @@
 FC_OBJECT (RGBA,		FcTypeInteger,	NULL)
 FC_OBJECT (SCALE,		FcTypeDouble,	NULL)
 FC_OBJECT (MINSPACE,		FcTypeBool,	NULL)
-FC_OBJECT (CHAR_WIDTH,		FcTypeInteger,	NULL)
+FC_OBJECT (CHARWIDTH,		FcTypeInteger,	NULL)
 FC_OBJECT (CHAR_HEIGHT,		FcTypeInteger,	NULL)
 FC_OBJECT (MATRIX,		FcTypeMatrix,	NULL)
 FC_OBJECT (CHARSET,		FcTypeCharSet,	FcCompareCharSet)
diff -ur fontconfig-2.11.1.orig/src/fcobjshash.gperf fontconfig-2.11.1/src/fcobjshash.gperf
--- fontconfig-2.11.1.orig/src/fcobjshash.gperf	2017-11-27 19:46:28.708675323 -0800
+++ fontconfig-2.11.1/src/fcobjshash.gperf	2017-11-27 19:47:14.172539462 -0800
@@ -44,7 +44,7 @@
 "rgba",FC_RGBA_OBJECT
 "scale",FC_SCALE_OBJECT
 "minspace",FC_MINSPACE_OBJECT
-"charwidth",FC_CHAR_WIDTH_OBJECT
+"charwidth",FC_CHARWIDTH_OBJECT
 "charheight",FC_CHAR_HEIGHT_OBJECT
 "matrix",FC_MATRIX_OBJECT
 "charset",FC_CHARSET_OBJECT
diff -ur fontconfig-2.11.1.orig/src/fcobjshash.h fontconfig-2.11.1/src/fcobjshash.h
--- fontconfig-2.11.1.orig/src/fcobjshash.h	2017-11-27 19:46:28.708675323 -0800
+++ fontconfig-2.11.1/src/fcobjshash.h	2017-11-27 19:47:37.808468830 -0800
@@ -275,7 +275,7 @@
 #line 26 "fcobjshash.gperf"
       {(int)(long)&((struct FcObjectTypeNamePool_t *)0)->FcObjectTypeNamePool_str40,FC_WIDTH_OBJECT},
 #line 47 "fcobjshash.gperf"
-      {(int)(long)&((struct FcObjectTypeNamePool_t *)0)->FcObjectTypeNamePool_str41,FC_CHAR_WIDTH_OBJECT},
+      {(int)(long)&((struct FcObjectTypeNamePool_t *)0)->FcObjectTypeNamePool_str41,FC_CHARWIDTH_OBJECT},
 #line 48 "fcobjshash.gperf"
       {(int)(long)&((struct FcObjectTypeNamePool_t *)0)->FcObjectTypeNamePool_str42,FC_CHAR_HEIGHT_OBJECT},
 #line 44 "fcobjshash.gperf"
